docs/book/src/developers/development.md

```diff
@@ -75,6 +75,7 @@
 10. Install [pre-commit framework](https://pre-commit.com/#installation)
    - `brew install pre-commit` Or `pip install pre-commit`. Installs pre-commit globally.
    - run `pre-commit install` at the root of the project to install pre-commit hooks to read `.pre-commit-config.yaml`
+   - run `pre-commit install --hook-type pre-push` to install pre-push hooks.
```

Comment:
I don't think you need this necessarily. The regular install command seems to install all valid stage types:
```
% pre-commit uninstall
pre-commit uninstalled
pre-push uninstalled

% pre-commit install
pre-commit installed at .git/hooks/pre-commit
pre-commit installed at .git/hooks/pre-push

```

Reply:
Since I rebased local/main with the new changes, pre-commit seemed to miss triggering `pre-push` stage. 
Reinstalling pre-push stage using `pre-commit install --hook-type pre-push` solved the issue for me.

So I think, as you said, a fresh pre-commit install will install all hooks listed in `default_install_hook_types`. Ref: https://pre-commit.com/#pre-commit-configyaml---top-level 

I think I can drop this note in the doc.